### PR TITLE
feat(console,ui,phrases,schemas): update favicon and html title

### PR DESCRIPTION
**Pull request by simeng-li**
### dynamically update favicon and html title

This PR includes the following updates:

*schemas*:
 - add an optional favicon URL field to the SIE branding settings

*console*:
 - add the favicon InputField in SIE settings page
image

*ui*:
 - dynamically update the HTML title based on the current page path
    - sign-in for all sign-in flow pages 
        image

     - create account for all sign-up flow pages
       image

     - reset password for reset password flow pages
        image

     - use logto as default i.e. 404 pages

  - read the favicon settings from SIE, and dynamically set the HTML favicon and apple-touch-icon. Use the local logto icon assets as a fallback.
    
image
image

## Testing

test locally

## Checklist

- [x] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

File: .changeset-staged/eight-shoes-look.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+"@logto/console": patch
+"@logto/phrases": patch
+"@logto/phrases-ui": patch
+"@logto/schemas": patch
+"@logto/ui": patch
```
Comment (gao-sun) [inline .changeset-staged/eight-shoes-look.md:6]: should be minor since it's a new feature?

File: packages/ui/src/containers/AppContent/index.tsx [hunk 2]
```diff
@@ -1,12 +1,25 @@
-import { Outlet } from 'react-router-dom';
+import { useEffect } from 'react';
```
Comment (gao-sun) [inline packages/ui/src/containers/AppContent/index.tsx:1]: the above favicon file uses the old logo. should be the new one?

**Review by gao-sun — APPROVED** [review]
no blocker

File: .changeset-staged/eight-shoes-look.md (hunk 1 above)
Comment (simeng-li) [reply .changeset-staged/eight-shoes-look.md]: updated

**Pull request merged**